Description: Drop sphinx_panels, not provided in Debian
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2021-05-10
Forwarded: not-needed

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -85,7 +85,7 @@
     "nbsphinx",
 #    "sphinx_autosummary_accessors",
     "sphinx.ext.linkcode",
-    "sphinx_panels",
+#    "sphinx_panels",
     "sphinxext.opengraph",
     "sphinx_copybutton",
     "sphinxext.rediraffe",
